src/components/Input/__tests__/__snapshots__/Input.spec.js.snap

```diff
@@ -78,6 +78,30 @@ exports[`Input renders input with error 1`] = `
 </div>
 `;
 
+exports[`Input renders input without name 1`] = `
+<div
+  class="text--input-regular text--input-top sc-bdVaJa cBKMGA"
+>
+  
+  <div
+    class="sc-htpNat eXpulu"
+  >
+    <input
+      aria-labelledby="null__label"
+      class="sc-bxivhb hBvKYg"
+      id="null__input"
```

Comment:
I think we should omit the `name` prop from being `sluggified` if its value is falsy.

Also, I am concerned that by not passing an explicit `name` prop we may generate non-unique `id` attributes for multiple `input` elements. Any thoughts on how we can mitigate that potential issue?

Reply:
Yeah, about that. Why do we need to auto generate ids? Maybe we could just accept it as optional prop. We could do the same with label... What do you think about it?

Reply:
We are generating the ids for use with the `[aria-labelledby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute)` attribute. It's fine with me to make `name` an optional prop. In the case that it is not passed, we just need to ensure that we don't pass attributes that are relying on it.